quartz/components/Body.tsx

```diff
@@ -4,7 +4,7 @@ import clipboardStyle from "./styles/clipboard.scss"
 import { QuartzComponent, QuartzComponentConstructor, QuartzComponentProps } from "./types"
 
 const Body: QuartzComponent = ({ children }: QuartzComponentProps) => {
-  return <div id="quartz-body">{children}</div>
+  return <main id="quartz-body">{children}</main>
```

Comment:
i think main should be for `quartz-root`, and this can be either a section or a div

Reply:
No, 

> The content inside the <main> element should be unique to the document. It should not contain any content that is repeated across documents such as sidebars, navigation links, copyright information, site logos, and search forms.

So I have place it around Content.